src/studio/ui/SettingsDialog.java

```diff
@@ -13,13 +13,32 @@
 import static javax.swing.GroupLayout.PREFERRED_SIZE;
 
 public class SettingsDialog extends EscapeDialog {
+    private static class FontBox {
+        final JLabel nameLabel;
+        final JTextField nameTextField;
+        final JLabel sizeLabel;
+        final JSpinner sizeSpinner;
+        FontBox(String qual, Font current) {
+            nameLabel = new JLabel(qual + " font");
+            nameTextField = new JTextField(current.getName());
+            sizeLabel = new JLabel(qual + " font size ");
+            SpinnerNumberModel model = new SpinnerNumberModel(current.getSize(), 6, 72, 1);
+            sizeSpinner = new JSpinner(model);
+        }
+
+        Font getFont() {
+            return new Font(nameTextField.getText(), Font.PLAIN, (int)sizeSpinner.getValue());
+        }
+    }
```

Comment:
Obviously it's very good starting point (it's much better than nothing :) ).
FYI: In the future, I thought to incorporate: https://gitlab.com/dheid/fontchooser

Reply:
Yep, that what regular user expects to see when choosing font

Reply:
I realised that the JFreeChart (which we already have) has font chooser - see how it is called at org.jfree.chart.editor.DefaultTitleEditor.attemptFontSelection. Do you want to include this font chooser here?